### PR TITLE
Add turba LDAP addressbook

### PR DESCRIPTION
**Pull request by mrmarkuz**
https://community.nethserver.org/t/installing-horde-groupware/7292/49

Add configuration for both LDAP/AD to have a shared LDAP addressbook in horde turba.

### CODE REVIEW

**Comment by DavidePrincipi** [comment]
@mrmarkuz can you ask a code review on this PR? Do you have enough rights?

Please add a reference to the nethserver community discussion or the GitHub issue!

BTW @mtraeumner has admin rights on this repo

File: nethserver-horde.spec [hunk 1]
```diff
@@ -37,3 +37,7 @@ rm -rf %{buildroot}
 %dir %{_nseventsdir}/%{name}-update
 
 %changelog
+* Wed Mar 21 2018 Markus Neuberger <dev@markusneuberger.at> - 0.0.1-1
```
Comment (DavidePrincipi) [inline nethserver-horde.spec]: Do not alter the `%changelog` section manually during development. Usually changelog entries are added with `release-tag`, when a new RPM is released. See

http://docs.nethserver.org/projects/nethserver-devel/en/v7/building_rpms.html#creating-a-release-tag

**Review by DavidePrincipi — APPROVED** [review]

File: nethserver-horde.spec (hunk 1 above)
Comment (mrmarkuz) [reply nethserver-horde.spec]: Thanks for review and infos. It seems to take some time until I fully understand this process. Should I do another PR without the changelog entries?

Comment (DavidePrincipi) [reply nethserver-horde.spec]: No, you should push a new commit to this PR that fixes the line above. You could even edit the file from GitHub web interface itself ;)

In the future I suggest the following git commands workflow:

- `git remote add mrmarkuz <GITURL>`
- `git fetch origin`
- `git checkout -b change-proposal origin/master`
- edit files
- `git commit`
- `git push mrmarkuz`

Then access the origin repo on github and open the PR from your newly pushed `change-proposal` branch. In short: do not use "master" as origin branch, it avoids confusion if you open multiple PRs or your PR is "squashed".

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**